### ISSUE TITLE
The Prometheus metric status="parse-error" is wrong

### ISSUE THREAD

**Issue by liorfranko** [F5Networks/k8s-bigip-ctlr#931]
#### Description
The Prometheus metric status="parse-error" is wrong.
When creating a faulty configmap for a VIP on the F5, for example with a wrong partition, on the bigip-ctlr logs we see logs "Error parsing ConfigMap", but in Prometheus the metric status="parse-error" is not equal 1, what happens is that the metric for the faulty configmap is not generated at all.

#### Kubernetes Version
1.11.6

#### Controller Version
1.8.1

#### BIG-IP Version
12.1.2

#### Diagnostic Information
Prometheus metrics when the configmap is valid:
```
bigip_monitored_services{name="lp-nodeplayground-app",namespace="qa",status="parse-error"} 0
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="port-not-found"} 0
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="service-not-found"} 0
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="success"} 1
```

When I edit the configmap and use an invalid partition, this is the error from the bigip-ctlr logs:
`"Error parsing ConfigMap qa_lp-nodeplayground-app-vip-3000"`
But there aren't any metrics in Prometheus for name="lp-nodeplayground-app" nor name="qa_lp-nodeplayground-app-vip-3000"

#### Expected behavior when parse error occurs
```
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="parse-error"} 1
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="port-not-found"} 0
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="service-not-found"} 0
bigip_monitored_services{name="qa_lp-nodeplayground-app-vip-3000",namespace="qa",status="success"} 0
```

**Comment by mdditt2000** [comment]
CONTCNTR-864

**Comment by ssurenr** [comment]
This is fixed in 1.12.0
